project_organization.md

```diff
@@ -55,6 +55,27 @@ OTIAB members have the following primary responsibilities:
 - Provide feedback about possible priorities and/or specific proposals from the OTSC
 - Represent OpenTracing’s interests within their own organizations
 
+## Documentation Working Group
+
+The OpenTracing community is growing rapidly, moving from a tight-knit group of early adopters to a larger group of developers. In order to support this transition, the Documentation Working Group is tasked with codifying the existing knowledge base into proper documentation which can be easily accessed via the OpenTracing website.
```

Comment:
Seems like the "documentation working group" is something that will always exist (i.e., it's not transitional). This paragraph feels like a great thing for the PR description, but for this actual `.md` file I'd rather just describe what the doc working group **is** without referring to the point in time when this paragraph happened to be added to the `.md` file. E.g., maybe rewrite to be just the following:

```
The Documentation Working Group codifies and maintains the existing knowledge base as  proper documentation which can be easily discovered via the OpenTracing website.
```
?